lib/rucio/core/did.py

```diff
@@ -2734,6 +2734,7 @@ def set_status(scope, name, session=None, **kwargs):
     :param kwargs:  Keyword arguments of the form status_name=value.
     """
     statuses = ['open', ]
+    reevaluate_close_did = config_get_bool('core', 'reevaluate_close_did', raise_exception=False, default=False, session=session)
```

Comment:
I am wondering whether `reevaluate_close_did` has room for improvement. Maybe `reevaluate_closed_dids`, `reevaluate_dids_at_close` or `reevaluate_dids_at_closure`? I don’t really like any of them.

Reply:
Of course, none of them are good. They are all snake case, it should be camel case :smile: !!
Seriously, let's just decide on one. My preferred one is `reevaluate_dids_at_close`

Reply:
`...at_close` implies that it only happens when the DID is closed.

`reevaluate_closed_dids` is what you mean, right?

Reply:
I’m not sure I can agree with you, Mario. The re-evaluation indeed is supposed to happen once: at the time the dataset is closed. Which is correct. In turn, `reevaluate_closed_dids` implies a continuous operation.

Reply:
aha! then i misunderstood and `reevaluate_dids_at_close` is indeed the right one